src/styles/variables.scss

```diff
@@ -0,0 +1,31 @@
+//colors
+$mint: #3AAFA9;
+$dark-green: #2B7A78;
+$light-green: #DEF2F1;
+$bg-color-light: #FEFFFF;
+$dark-grey: #17252A;
+
+//fonts
+$title: "Fira Sans", sans-serif;
+$main-font: "Source Sans Pro", sans-serif;
+
+//font sizes
+$font-size-14-mobile: 14px;
+$font-size-18-mobile: 18px;
+$font-size-24-title-mobile: 24px;
```

Comment:
what if the title size is changed to 26px for example, will you change the variable name also?

Reply:
i renamed the variables for font sizes, so they are more general and not tied to size (s - small, m - medium, l - large)

Reply:
cause changing the variable name will case to change code in each place we use it.